src/main/java/org/app/TransactionsController.java

```diff
@@ -21,15 +22,18 @@
 
 @RestController
 class TransactionsController {
-    @GetMapping("transactions")
+    @Autowired
+    private TransactionsHandler transactionsHandler;
+
+    @GetMapping("/transactions")
     public List<Transaction> getTransactions() {
-        return TransactionsHandler.getInstance().getAllTransactions();
+        return transactionsHandler.getAllTransactions();
     }
 
     @PostMapping("/transactions")
```

Comment:
If both GET and POST methods are using the same url (which is perfect), why not making that string a constant?

Reply:
Fixed